Packs/CiscoWSA/Integrations/CiscoWSAV2/CiscoWSAV2.py

```diff
@@ -0,0 +1,1663 @@
+                            "yt_cats": {youtube_categories_action: youtube_categories},
+                            "custom_cats": {
+                                custom_categories_action: custom_categories
+                            },
+                            "uncategorized_url": uncategorized_url,
+                            "update_cats_action": update_categories_action,
+                            "content_rating": {
+                                "status": content_rating_status,
+                                "action": content_rating_action,
+                            },
+                            "safe_search": {
+                                "status": safe_search_status,
+                                "unsupported_safe_search_engine": unsupported_safe_search_engine,
+                            },
+                        },
+                    }
+                ]
+            }
+        )
+        return self._http_request(
+            "PUT",
+            f"{V3_PREFIX}/web_security/access_policies",
+            json_data=data,
+            resp_type="response",
+            ok_codes=[HTTPStatus.NO_CONTENT],
+        )
+
+    def access_policy_applications_update(
+        self,
+        policy_name: str,
+        application: str,
+        action: str,
+        values: dict[str, Any],
+        settings_status: str,
+    ) -> Response:
+        """
+        Update access policy's applications settings.
+
+        Args:
+            policy_name (str): Policy name to update.
+            application (str): Application to update.
+            action (str): Action to perform on values.
+            values (dict[str, Any]): Values to perform action on.
+            settings_status (str): Settings status for the policy.
+
+        Returns:
+            Response: API response from Cisco WSA.
+        """
+        data = {
+            "access_policies": [
+                {
+                    "policy_name": policy_name,
+                    "avc": {
+                        "applications": {
+                            application: {action: values}
+                            if action == "block"
+                            else {action: {value: {} for value in values}},
+                        },
+                        "state": settings_status,
+                    },
+                }
+            ]
+        }
+
+        return self._http_request(
+            "PUT",
+            f"{V3_PREFIX}/web_security/access_policies",
+            json_data=data,
+            resp_type="response",
+            ok_codes=[HTTPStatus.NO_CONTENT],
+        )
+
+    def access_policy_objects_update(
+        self,
+        policy_name: str,
+        object_type: str | None,
+        object_action: str | None,
+        object_values: List[str] | None,
+        block_custom_mime_types: List[str] | None,
+        http_or_https_max_object_size_mb: int | None,
+        ftp_max_object_size_mb: int | None,
+    ) -> Response:
```

Comment:
Why the return type here is  'Response' and in other request functions it is  'dict[str, Any]'?
We need to be consistent about that. 🕵🏼‍♀️